IdentityCore/src/telemetry/request_telemetry/MSIDLastRequestTelemetrySerializedItem.m

```diff
@@ -74,11 +75,17 @@ - (NSString *)serializeErrorsInfoWithCurrentStringSize:(NSUInteger)startLength
         
         // Only add error info into string if the resulting string smaller than 4KB
         if ([currentFailedRequest lengthOfBytesUsingEncoding:NSUTF8StringEncoding] +
-                [currentErrorMessage lengthOfBytesUsingEncoding:NSUTF8StringEncoding] + startLength < 4000)
+                [currentErrorMessage lengthOfBytesUsingEncoding:NSUTF8StringEncoding] + startLength < [MSIDCurrentRequestTelemetrySerializedItem telemetryStringSizeLimit])
         {
             failedRequestsString = [currentFailedRequest stringByAppendingString:failedRequestsString];
             errorMessagesString = [currentErrorMessage stringByAppendingString:errorMessagesString];
         }
+        else
+        {
+            [self addRemainingErrorsToUnserializedTelemetry:lastIndex];
+            
+            return [NSString stringWithFormat:@"%@|%@", failedRequestsString, errorMessagesString];
```

Comment:
we aren't sending actual error message right?

Reply:
No, we're not. The errors that are passed to LastRequestTelemetry are modified by [error msidServerTelemetryErrorString].